### ISSUE TITLE
diff-pdf failing to build on macOS 10.15.7

### ISSUE THREAD

**Issue by zlscherr** [Homebrew/homebrew-core#66385]
# Bug report

**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [X] ran `brew update` and can still reproduce the problem?
- [X] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [X] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

## What you were trying to do (and why)

I tried to install diff-pdf via brew install diff-pdf

## What happened (include command output)

the bottle for diff-pdf could not found, and then it failed to build from source due to not being able to find poppler-cairo.  I have both of those currently installed with homebrew. 

<details>
  <summary>Command output</summary>
  <pre>

==> Downloading https://homebrew.bintray.com/bottles/diff-pdf-0.4.1_8.catalina.bottle.tar.gz
#=#=-#  #
curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "diff-pdf"
Download failed: https://homebrew.bintray.com/bottles/diff-pdf-0.4.1_8.catalina.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://github.com/vslavik/diff-pdf/releases/download/v0.4.1/diff-pdf-0.4.1.tar.gz
Already downloaded: /Users/zscherr/Library/Caches/Homebrew/downloads/ed587a20534d95120971bb9dd050b7bbf84210d189ebefda91a082f11512205e--diff-pdf-0.4.1.tar.gz
==> ./configure --prefix=/usr/local/Cellar/diff-pdf/0.4.1_8
Last 15 lines from /Users/zscherr/Library/Logs/Homebrew/diff-pdf/01.configure:
checking for style of include used by make... GNU
checking dependency style of clang++... none
checking for pkg-config... /usr/local/Homebrew/Library/Homebrew/shims/mac/super/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for POPPLER... no
configure: error: Package requirements (poppler-cairo >= 0.10 poppler-glib >= 0.10 cairo-pdf) were not met:

No package 'poppler-cairo' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables POPPLER_CFLAGS
and POPPLER_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

READ THIS: https://docs.brew.sh/Troubleshooting

  </pre>
</details>

## What you expected to happen

I expected diff-pdf to install correctly.

## Step-by-step reproduction instructions (by running `brew install` commands)

brew install diff-pdf

The gist file is located here: https://gist.github.com/zlscherr/8d885f4da1ea1eeb624f58e7f19fb2e7

**Comment by fxcoudert** [comment]
We are in the process of rebuilding it against a newer version of poppler: https://github.com/Homebrew/homebrew-core/pull/66369

**Comment by ko5tas** [comment]
got the same issue when trying to upgrade: 

```
$ brew upgrade                                                                                                                                                               cluster.build.ngiris.io ⎈  22:53:03 GMT
==> Upgrading 1 outdated package:
diff-pdf 0.4.1_7 -> 0.4.1_8
==> Upgrading diff-pdf 0.4.1_7 -> 0.4.1_8
==> Downloading https://homebrew.bintray.com/bottles/diff-pdf-0.4.1_8.big_sur.bottle.tar.gz
##O=#  #
curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "diff-pdf"
Download failed: https://homebrew.bintray.com/bottles/diff-pdf-0.4.1_8.big_sur.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://github.com/vslavik/diff-pdf/releases/download/v0.4.1/diff-pdf-0.4.1.tar.gz
Already downloaded: /Users/kostas/Library/Caches/Homebrew/downloads/ed587a20534d95120971bb9dd050b7bbf84210d189ebefda91a082f11512205e--diff-pdf-0.4.1.tar.gz
==> ./configure --prefix=/usr/local/Cellar/diff-pdf/0.4.1_8
Last 15 lines from /Users/kostas/Library/Logs/Homebrew/diff-pdf/01.configure:
checking for style of include used by make... GNU
checking dependency style of clang++... none
checking for pkg-config... /usr/local/Homebrew/Library/Homebrew/shims/mac/super/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for POPPLER... no
configure: error: Package requirements (poppler-cairo >= 0.10 poppler-glib >= 0.10 cairo-pdf) were not met:

No package 'poppler-cairo' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables POPPLER_CFLAGS
and POPPLER_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

READ THIS: https://docs.brew.sh/Troubleshooting

These open issues may also help:
diff-pdf failing to build on macOS 10.15.7 https://github.com/Homebrew/homebrew-core/issues/66385
```

**Comment by ko5tas** [comment]
got the same issue when trying to upgrade: 

```
$ brew upgrade 
==> Upgrading 1 outdated package:
diff-pdf 0.4.1_7 -> 0.4.1_8
==> Upgrading diff-pdf 0.4.1_7 -> 0.4.1_8
==> Downloading https://homebrew.bintray.com/bottles/diff-pdf-0.4.1_8.big_sur.bottle.tar.gz
##O=#  #
curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "diff-pdf"
Download failed: https://homebrew.bintray.com/bottles/diff-pdf-0.4.1_8.big_sur.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading https://github.com/vslavik/diff-pdf/releases/download/v0.4.1/diff-pdf-0.4.1.tar.gz
Already downloaded: /Users/kostas/Library/Caches/Homebrew/downloads/ed587a20534d95120971bb9dd050b7bbf84210d189ebefda91a082f11512205e--diff-pdf-0.4.1.tar.gz
==> ./configure --prefix=/usr/local/Cellar/diff-pdf/0.4.1_8
Last 15 lines from /Users/kostas/Library/Logs/Homebrew/diff-pdf/01.configure:
checking for style of include used by make... GNU
checking dependency style of clang++... none
checking for pkg-config... /usr/local/Homebrew/Library/Homebrew/shims/mac/super/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for POPPLER... no
configure: error: Package requirements (poppler-cairo >= 0.10 poppler-glib >= 0.10 cairo-pdf) were not met:

No package 'poppler-cairo' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables POPPLER_CFLAGS
and POPPLER_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

READ THIS: https://docs.brew.sh/Troubleshooting

These open issues may also help:
diff-pdf failing to build on macOS 10.15.7 https://github.com/Homebrew/homebrew-core/issues/66385
```

```
$ sw_vers 
ProductName:	macOS
ProductVersion:	11.0.1
BuildVersion:	20B50
```

**Comment by asalsano** [comment]
having this issue also when trying to install diff-pdf

**Comment by chris-xstem** [comment]
still having this issue

<img width="817" alt="image" src="https://user-images.githubusercontent.com/18240977/102574805-a5258c80-412c-11eb-8176-8bec285351f2.png">

**Comment by fxcoudert** [comment]
Fixed with 0083684ae91f0da414dc76c46f974610a46bd13a
